app/src/main/java/com/github/onlynotesswent/model/note/Note.kt

```diff
@@ -150,5 +165,17 @@ data class Note(
    * @property userId The unique identifier of the user who posted the comment.
    * @property content The text content of the comment.
    */
-  data class Comment(val commentId: String, val userId: String, val content: String)
+  data class Comment(
+      val commentId: String,
+      val userId: String,
+      val userName: String,
+      val content: String,
+      val creationDate: Timestamp,
+      val editedDate: Timestamp
+  ) {
+    // Method to check if creationDate is equal to editedDate
+    fun isUnedited(): Boolean {
```

Comment:
As this function designed to be public (used in EditNote), document it with /** */